modal/config.py

```diff
@@ -171,6 +171,7 @@ class _Setting(typing.NamedTuple):
     "default_cloud": _Setting(None, transform=lambda x: x if x else None),
     "worker_id": _Setting(),  # For internal debugging use.
     "restore_state_path": _Setting("/opt/modal/restore-state.json"),
+    "container_id": _Setting(),
```

Comment:
Maybe remove this setting?

Reply:
Removed and re-added: it's good to have `MODAL_CHECKPOINT_ID` populated for debugging with users.